src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,14 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
```

Comment:
There's an unnecessary empty line after the main method signature. According to the checklist, such empty lines should be removed to keep the code clean.